install/updates/73-certmap.update

```diff
@@ -0,0 +1,27 @@
+# Configuration for Certificate Identity Mapping
+dn: cn=certmap,cn=ipa,cn=etc,$SUFFIX
+default:objectclass: top
+default:objectclass: nsContainer
+default:objectclass: ipaConfigObject
+default:objectclass: ipaCertMapContainer
+default:objectclass: ipaCertMapConfigObject
+default:cn: certmap
+default:ipaconfigstring: CertMapVersion 1
+default:ipacertmapversion: 1
+default:ipaCertMapPromptUsername: FALSE
+
+dn: cn=certmaprules,cn=certmap,cn=ipa,cn=etc,$SUFFIX
+default:objectclass: top
+default:objectclass: nsContainer
+default:cn: certmaprules
+
+# Certificate Identity Mapping Administrators
+dn: cn=Certificate Identity Mapping Administrators,cn=privileges,cn=pbac,$SUFFIX
+default:objectClass: top
+default:objectClass: groupofnames
+default:objectClass: nestedgroup
+default:cn: Certificate Identity Mapping Administrators
+default:description: Certificate Identity Mapping Administrators
```

Comment:
Hi @HonzaCholasta 
I simply had a look at install/updates/60-trusts.update and did the same, i.e. define new containers and privileges in the same file.
I will keep your note for future reference though.